reconcile/gitlab_fork_compliance.py

```diff
@@ -84,20 +83,12 @@ def check_branch(self):
         return self.OK
 
     def check_bot_access(self):
-        # The bot needs access to the fork project
-        try:
-            self.src = GitLabApi(
-                self.instance,
-                project_id=self.mr.source_project_id,
-                settings=self.settings,
-            )
-            project_bot = self.src.project.members.get(self.gl_cli.user.id)
-        except GitlabGetError:
```

Comment:
I assume this is not an issue, but could this still be thrown on `self.src.get_project_maintainers()`?

Reply:
Good catch! That makes the change even smaller....